src/cmd/linuxkit/pkglib/index.go

```diff
@@ -0,0 +1,49 @@
+package pkglib
+
+import (
+	"fmt"
+
+	"github.com/linuxkit/linuxkit/src/cmd/linuxkit/registry"
+	"github.com/linuxkit/linuxkit/src/cmd/linuxkit/util"
+)
+
+func (p Pkg) Index(bos ...BuildOpt) error {
```

Comment:
Can you please add comment for this exported function?

Reply:
Done